### DISCUSSION TITLE
How to move a branch on another branch?

### DISCUSSION THREAD

**Discussion by timmak** [gitext-rs/git-stack#298] [Q&A] [answered]
Given I branch A and find an issue which means I want to create a new branch B, I branch off main, but now I would like to move A from main onto B.  

**From**
```
        - A    - B
main --/------/
```

**To**
```
       - B- A 
main--/
```

**Comment by epage** [comment, marked as answer]
I believe it would roughly be `git stack --base A --onto B`
